update-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# find `ablyAssetTrackingLibraryVersion = "current_version"` and replace `current_version` with $1
+# script starts searching the pattern in project root directory recursively
```

Comment:
I'd add an example usage of the script to this comment :wink: And maybe let's explain what the script does on a higher level (what it changes from the project point of view) because the current explanation is very low level :thinking:

Reply:
explained here https://github.com/ably/ably-asset-tracking-swift/pull/188/commits/cf5138baacf80b61abb0aaefde19997189520eec